### DISCUSSION TITLE
最近vercel好像被墙了，大家有什么好的解决方案吗？

### DISCUSSION THREAD

**Discussion by dongwa** [Binaryify/NeteaseCloudMusicApi#1612] [Q&A] [answered]
最近发现在vercel 上部署的服务全都无法访问了，好像是vercel被墙了，大家有什么好的解决方案吗？

**Comment by whyqiuqiu** [comment]
我用的腾讯云

**Reply by dongwa** [reply]
腾讯云不要钱嘛，vercel 可以白嫖

**Reply by ZJRui** [reply]
腾讯云 exprss的node版本是12， 还不让改
![image](https://github.com/Binaryify/NeteaseCloudMusicApi/assets/40123628/652ca8eb-acf7-4f65-94c7-515ba4c85957)

**Comment by Binaryify** [comment, marked as answer]
绑定国内域名，然后Function Region 设置为香港
<img width="1522" alt="image" src="https://github.com/Binaryify/NeteaseCloudMusicApi/assets/12221718/69643ee9-0240-4fb7-9e08-e7acf220d675">

**Comment by vaynevayne** [comment]
<img width="862" alt="image" src="https://github.com/Binaryify/NeteaseCloudMusicApi/assets/124666577/1b5ec27c-e814-4c22-adef-514158c5ecce">

凉凉
